roles/openldap/templates/password_policy_util.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
```

Comment:
Yes its intended to be used outside ansible as a helper script to unlock users/revoke password policy  of users etc.